CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## 1.1.1 - [#5](https://github.com/openfisca/extension-template/pull/5)
+
+* Technical improvement: adapt to version `17.0.0` of Openfisca-Core and version `1.2.4` of Country-Template
```

Comment:
I agree that in the specific context of the extension template, the main point is to adapt to the latest version of the depending packages. I would appreciate that more detail is given, but this is in no way blocking for me here. It's not like anybody actually cares about the extension template beyond when starting up a new country package :wink: